### PR TITLE
Sticky footer [FOGL-2506]

### PR DESCRIPTION
**Pull request by MonikaSharma06**

### CODE REVIEW

File: src/app/components/layout/footer/footer.component.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    text-align: center;
+    border-top: 1px solid #e7e7e7;
+    padding: 2rem 1.5rem 2rem !important;
```
Comment (praveen-garg) [inline src/app/components/layout/footer/footer.component.css]: can we make it `1rem 1.5rem 1rem !important;`?

**Review by praveen-garg — CHANGES_REQUESTED** [review]
screenshot 2019-02-21 at 2 10 08 pm

why the version label is missing which we added recently?
why these blue dropdown icon appears on top of footer instead of getting hidden behind?
can we make copyright+ version stuff center of main container, instead of whole viewport.

File: src/app/app.component.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .column.is-offset-2.is-10 {
   height: 100%;
+  padding-bottom: 120px;
```
Comment (praveen-garg) [inline src/app/app.component.css]: why this padding added? and in px?!

File: src/app/components/layout/footer/footer.component.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    height: 120px;
```
Comment (praveen-garg) [inline src/app/components/layout/footer/footer.component.css]: why this height added? and in px?!

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by mshariq-nerd** [comment]
LGTM! other then  pending `copyright+ version stuff center of main container, instead of whole viewport`

**Comment by MonikaSharma06** [comment]
Closing in favour of [https://github.com/foglamp/foglamp-gui/pull/564](https://github.com/foglamp/foglamp-gui/pull/564)

**Pull request closed without merging**